### ISSUE TITLE
Registering new user with App client secret fails

### ISSUE THREAD

**Issue by whatnick** [capless/warrant#67]
I am testing the new support for App Secrets and the register function fails for new users using an App Client with secret and ADMIN_NO_SRP_AUTH configured.

```
u = Cognito(cognito_pool,cognito_app_client,client_secret=cognito_app_secret)
u.register(username, password, email=email)
```

Error:

`An error occurred (NotAuthorizedException) when calling the SignUp operation: Unable to verify secret hash for client`

The use case is to create new users when a purchase is performed via Shopify or other 3rd party eCommerce platform.

**Comment by wesdyoung** [comment]
Yes, I have been experiencing this issue too.  The solution is to fork the project and edit the register and a number of other functions that do not add in ClientSecret into the request.

When I get a chance I will create a PR for this.

**Comment by armicron** [comment]
Should be fixed in #70.
